src/usr/local/buildpack/tools/v2/kustomize.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+function install_tool () {
+  local versioned_tool_path
+  local file
+  local arch=linux_amd64
```

Comment:
please prepare for arm64, checkout the open pr

Reply:
nice, fixed!